xarray/core/dataarray.py

```diff
@@ -3426,11 +3427,12 @@ def sortby(
 
     def quantile(
         self,
-        q: Any,
-        dim: Hashable | Sequence[Hashable] | None = None,
-        interpolation: str = "linear",
+        q: ArrayLike,
+        dim: str | Sequence[Hashable] | None = None,
+        method: str = "linear",
```

Comment:
Can use `Literal["linear", etc]` here instead of `str`.

Reply:
I feel that's a bit over the top - can I get away without it?

Reply:
Agree it would be cool, definitely fine without out too! :)